File: app/src/main/java/com/chess/cryptobot/content/pairs/PairsPreferences.java
package com.chess.cryptobot.content.pairs;

import android.content.Context;

import com.chess.cryptobot.R;
import com.chess.cryptobot.content.Preferences;

class PairsPreferences extends Preferences {
    PairsPreferences(Context context) {
        super(context);
    }

    @Override
    public String initPrefKey(Context context) {
        return context.getResources().getString(R.string.pairs_pref_key);
    }
}

File: app/src/main/java/com/chess/cryptobot/content/pairs/PairsPreferences.kt
package com.chess.cryptobot.content.pairs

import android.content.Context

import com.chess.cryptobot.R
import com.chess.cryptobot.content.Preferences

internal class PairsPreferences(context: Context) : Preferences(context) {

    public override fun initPrefKey(context: Context): String {
        return context.resources?.getString(R.string.pairs_pref_key) ?: "pairs"
    }
}
